### ISSUE TITLE
可以做一个用jdk8编译openj9的项目吗？

### ISSUE THREAD

**Issue by MCredbear** [ibmruntimes/openj9-openjdk-jdk8#306]
由于某些原因，我必须使用jdk8

**Comment by JasonFengJ9** [comment]
Yes, you can https://github.com/eclipse/openj9/blob/master/doc/build-instructions/Build_Instructions_V8.md.

**Comment by MCredbear** [comment]
但是它好像不能适用于arm64架构

**Comment by JasonFengJ9** [comment]
That's correct, no `ARM64` for `OpenJ9` JDK8 yet.

Additional note: we have internal `ARM32` build with `JDK8` so some framework is there.
```
IBM J9 VM (build 2.9, JRE 1.8.0 Linux arm-32-Bit 20190522_417153 (JIT enabled, AOT enabled)
OpenJ9   - f73eb7f
OMR      - 2c58202
IBM      - 4972efe)
JCL - 20151231_01 based on Oracle jdk8u71-b15
```

**Comment by pshipton** [comment]
> so some framework is there.

in OpenJ9 itself, not in the openjdk8 extensions

**Comment by MCredbear** [comment]
那你们将来会做一个用jdk8来编译openj9的分支吗？

**Comment by JasonFengJ9** [comment]
I think you meant `OpenJ9 JDK8 aarch64`, right?
As per https://github.com/ibmruntimes/openj9-openjdk-jdk8/issues/304#issuecomment-495214383, we are going to build it though no promise for the timeline.
